library/Translate/src/Juvix/Desugar/Passes.hs

```diff
@@ -20,9 +20,12 @@ module Juvix.Desugar.Passes
   )
 where
 
+import Control.Lens hiding ((|>))
 import qualified Data.Set as Set
 import Juvix.Library
 import qualified Juvix.Library.Sexp as Sexp
+import qualified Juvix.Sexp.Structure as Struct
+import qualified Juvix.Sexp.Structure as Structure
```

Comment:
Why do we want different names for the same imported module?

Reply:
Just because `longname ^. Structure.lensName` I found was too long, however I do want to keep the explicitness for `Structure.Constructor`, `Structure.to<Name>` etc etc.

Reply:
probably not worth it for the extra line. It creates confusion. `Structure.lensName` is probably fine.

Reply:
Well the issue is, this cascades into many lines taking 3~ lines more... I might just make a lens module under it to import unqualified, as it's essentially a record lookup which shouldn't be `qualifeid` to begin with. How do others feel about modules re-exporting othes (people can import the lens module for unqualified, or have it qualified in the normal?)

Reply:
I've added a Lens module now that is unqualified, hope that is fine with everyone